src/styles/zindex.scss

```diff
@@ -10,3 +10,11 @@ $z-index-menu-logo: 4;
  * Modal Header
  */
 $z-index-modal-header: 1;
+
+/**
+ * PopupButton
+ */
+$z-popup-button: 1000;
```

Comment:
we need try to be consistently. why 1000?

Reply:
Ok, so told me my past experience.

Reply:
And yet, I suggest all pop-up components to assign a large index. And already inside them sequentially set the index

Reply:
it's ok in projects when nobody knows about z-indexes order. Usual it's in chaos. And to make sure people do such "hacks". If we agree to follow the rule write all z-index properties in one file, it can help to avoid chaos.